automation_tools/__init__.py

```diff
@@ -218,13 +217,6 @@ def setup_default_docker():
     # https://github.com/docker/docker/issues/2758
     manage_daemon('restart', 'docker', pty=(os_version >= 7))
 
-    # Check that things look good
-    run('docker ps')
```

Comment:
We can leave this just to make sure that docker is really installed? (may be with warn_only=True)